go/stellar/bpc.go

```diff
@@ -69,8 +65,17 @@ func (c *buildPaymentCache) AccountSeqno(mctx libkb.MetaContext,
 }
 
 func (c *buildPaymentCache) IsAccountFunded(mctx libkb.MetaContext,
-	accountID stellar1.AccountID) (bool, error) {
-	return isAccountFunded(mctx.Ctx(), c.remoter, accountID)
+	accountID stellar1.AccountID, bid stellar1.BuildPaymentID) (res bool, err error) {
+	fill := func() (interface{}, error) {
+		return isAccountFunded(mctx.Ctx(), c.remoter, accountID)
+	}
+	if !bid.IsNil() {
```

Comment:
what's the case when this is nil?

Reply:
In theory buildPayment can be called with no bid. Though deprecated. Also some back story I thought it was like `GetOutsideCurrencyPreference` which is used by buildRequest, but it's not.